<ui version="4.0" >
 <author>Inge Wallin &lt;inge@lysator.liu.se></author>
 <comment>This file is part of the KDE project
Copyright 2005 Inge Wallin &lt;inge@lysator.liu.se>

This library is free software; you can redistribute it and/or
modify it under the terms of the GNU Library General Public
License as published by the Free Software Foundation; either
version 2 of the License, or (at your option) any later version.

This library is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
Library General Public License for more details.

You should have received a copy of the GNU Library General Public License
along with this library; see the file COPYING.LIB.  If not, write to
the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
Boston, MA 02110-1301, USA.
</comment>
 <class>KoCsvImportWidget</class>
 <widget class="QWidget" name="KoCsvImportWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>475</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTabWidget" name="m_tabWidget" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="m_dataTab" >
      <attribute name="title" >
       <string>Data</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="textLabel1_3" >
           <property name="text" >
            <string>E&amp;ncoding:</string>
           </property>
           <property name="textFormat" >
            <enum>Qt::PlainText</enum>
           </property>
           <property name="buddy" >
            <cstring>comboBoxEncoding</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KComboBox" name="comboBoxEncoding" >
           <property name="editable" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>106</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="m_firstRowHeader" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>First row contains headers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_firstColHeader" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>First column contains headers</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_formatTab" >
      <attribute name="title" >
       <string>Format</string>
      </attribute>
      <layout class="QGridLayout" >
       <item rowspan="4" row="0" column="0" >
        <widget class="QGroupBox" name="m_delimiterBox" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string>Delimiter</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QRadioButton" name="m_radioComma" >
            <property name="text" >
             <string>Comma</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QRadioButton" name="m_radioSemicolon" >
            <property name="text" >
             <string>Semicolon</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" >
           <widget class="QRadioButton" name="m_radioOther" >
            <property name="text" >
             <string>Other</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QRadioButton" name="m_radioTab" >
            <property name="text" >
             <string>Tabulator</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QRadioButton" name="m_radioSpace" >
            <property name="text" >
             <string>Space</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2" >
           <widget class="QLineEdit" name="m_delimiterEdit" >
            <property name="maximumSize" >
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maxLength" >
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item rowspan="5" row="0" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>111</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2" >
        <widget class="QLabel" name="label2" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Text&amp;quote:</string>
         </property>
         <property name="buddy" >
          <cstring>m_comboQuote</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="3" colspan="2" >
        <widget class="QComboBox" name="m_comboQuote" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text" >
           <string>"</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>'</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>None</string>
          </property>
         </item>
        </widget>
       </item>
       <item rowspan="5" row="0" column="5" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>90</width>
           <height>121</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2" >
        <widget class="QLabel" name="label3" >
         <property name="text" >
          <string>Decimal symbol:</string>
         </property>
         <property name="buddy" >
          <cstring>m_decimalSymbol</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="4" >
        <widget class="QLineEdit" name="m_decimalSymbol" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maxLength" >
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="2" >
        <widget class="QLabel" name="label4" >
         <property name="text" >
          <string>Thousands separator:</string>
         </property>
         <property name="buddy" >
          <cstring>m_thousandsSeparator</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="4" >
        <widget class="QLineEdit" name="m_thousandsSeparator" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maxLength" >
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="3" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>211</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0" >
        <widget class="QCheckBox" name="m_ignoreDuplicates" >
         <property name="text" >
          <string>Ignore duplicate delimiters</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2" >
        <widget class="QLabel" name="label1" >
         <property name="text" >
          <string>&amp;Format:</string>
         </property>
         <property name="buddy" >
          <cstring>m_formatComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="3" colspan="2" >
        <widget class="KComboBox" name="m_formatComboBox" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_rangesTab" >
      <attribute name="title" >
       <string>Ranges</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="2" column="5" >
        <widget class="QPushButton" name="m_updateButton" >
         <property name="text" >
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>71</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" colspan="4" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>611</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="4" colspan="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>468</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="TextLabel3_2" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape" >
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text" >
          <string>Import columns:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="TextLabel3" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape" >
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text" >
          <string>Import lines:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3" >
        <widget class="QSpinBox" name="m_colEnd" />
       </item>
       <item row="1" column="2" >
        <widget class="QLabel" name="textLabel1_2" >
         <property name="text" >
          <string>to</string>
         </property>
         <property name="textFormat" >
          <enum>Qt::AutoText</enum>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QSpinBox" name="m_colStart" >
         <property name="value" >
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2" >
        <widget class="QLabel" name="textLabel1" >
         <property name="text" >
          <string>to</string>
         </property>
         <property name="textFormat" >
          <enum>Qt::AutoText</enum>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QSpinBox" name="m_rowStart" >
         <property name="value" >
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="0" column="3" >
        <widget class="QSpinBox" name="m_rowEnd" />
       </item>
       <item row="0" column="4" colspan="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>468</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="Q3Table" name="m_sheet" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="numRows" >
      <number>0</number>
     </property>
     <property name="numCols" >
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
  <customwidgets>
  <customwidget>
   <class>Q3Table</class>
   <extends>Q3Frame</extends>
   <header>q3table.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="global" >kcombobox.h</include>
  <include location="global" >klineedit.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
